### PR TITLE
Fix #1153 Removed obligatory transfer after evolve

### LINKED ISSUE #1153: Bug - Transferring Pokemon after evolve above desired IV
**Description:**
I've set the transfer_iv_threshold=90 and it works as expected when catching Pokémons, but somehow it is not applied after evolving.

**Steps to reproduce:**

1. Stack evolve
2. Check if pokémon above transfer_iv_threshold were transferred 

**Expected behavior:**
Only transfer Pokemon below transfer_iv_threshold, transfer_cp_threshold and transfer_cp_min_threshold

**Actual behavior:**
Transferring pokémon above transfer_iv_threshold

**Stacktrace (If it's a crash):**
17 Aug 18:34:08 [default: ProfileLoop] - Evolving SPEAROW before transfer
17 Aug 18:34:09 [default: ProfileLoop] - Going to transfer FEAROW with CP 834 and IV 93%; reason: Evolved for XP only
17 Aug 18:35:46 [default: ProfileLoop] - Evolving SPEAROW before transfer
17 Aug 18:35:47 [default: ProfileLoop] - Going to transfer FEAROW with CP 24 and IV 93%; reason: Evolved for XP only
17 Aug 18:35:49 [default: ProfileLoop] - Evolving GOLDEEN before transfer
17 Aug 18:35:50 [default: ProfileLoop] - Going to transfer SEAKING with CP 744 and IV 91%; reason: Evolved for XP only
17 Aug 18:37:25 [default: ProfileLoop] - Evolving STARYU before transfer
17 Aug 18:37:26 [default: ProfileLoop] - Going to transfer STARMIE with CP 1110 and IV 93%; reason: Evolved for XP only

**Version:**
0.6.0

**Operating System:**
Mac OS X

**Java version:**
java version "1.8.0_91"
Java(TM) SE Runtime Environment (build 1.8.0_91-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.91-b14, mixed mode)

Leave this sentence in your issue as proof that you have read and used this template.

### PR DESCRIPTION
**Pull request by FrancYescO**
Fixed issue: #1153

### CODE REVIEW

**Comment by svombergen** [comment]
You will fill your pokebag with useless pidgeys and rattatas with this change.

**Comment by FrancYescO** [comment]
No. 
This just affects Pokémon after the evolve
You will still have Candy/candytoevolve pidgeys as before

File: src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt [hunk 1]
```diff
@@ -69,37 +56,29 @@ class EvolvePokemon : Task {
                 }
             }
             else {
-                Log.yellow("Not using lucky egg")
+                Log.yellow("Starting stack evolve of $countEvolveStack pokemons witouth lucky egg")
```
Comment (Sieberkev) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt]: Minor typo!

File: src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt [hunk 2]
```diff
@@ -33,34 +33,21 @@ class EvolvePokemon : Task {
...
                 } else {
-                    it.value.sortedByDescending { it.cp }
-                }
-                // Release pokemon we wont be able to evolve anyway, because of lack of candy:
-                sorted.forEach {
-                    maxPossibleEvolves--
-                    if (maxPossibleEvolves < 0) {
-                        releasePokemon(it,"Not enough candy to save for evolve",ctx)
-                    }
+                    Log.red("${it.key} is in evolve list but is unevolvable")
```
Comment (Sieberkev) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt]: Will this output not be spammed a lot?

Comment (FrancYescO) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt]: it's what i want... means you are stupid if you put a max evolved pokemon in the list

**Comment by Sieberkev** [comment]
Nice, I really like it :)

No more forced/wrong transfers in the ProfileLoop, and i guess a bunch less chance to get the null response because the transfers are done in the BotLoop.

Just found a typo and a question about an output ^_^

File: src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt (hunk 2 above)
Comment (Sieberkev) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt]: Right :+1:

**Pull request merged**